src/logFormatting.js

```diff
@@ -3,8 +3,13 @@ import get from 'lodash/get';
 import flatten from 'lodash/flatten';
 
 export function getElapsedTime(req) {
-    const diff = process.hrtime(req._startAt); // eslint-disable-line no-underscore-dangle
-    return (diff[0] * 1e3) + (diff[1] * 1e-6);
+    const startAt = get(req, '_startAt');
```

Comment:
I sohuld really check all the formatting functions to make sure they won't fail if the request object isn't formed correctly, I'll come back to that in a non-hotfix though.